### PR TITLE
Fix Pacman map generation

### PR DESCRIPTION
**Pull request by S2dentik**
Instead of crashing when Generator doesn't find a valid file
to generate the pacman map from, run pacman with the prototype,
default map.

### CODE REVIEW

**Comment by thelvis4** [comment]
How can we test this?

**Comment by S2dentik** [comment]
Go to Taylor folder and type these commands:

```
git checkout argumentsProcessing
git cherry-pick 9d39935154017cc81704035329457ea46f4fbb57
make install
```

Wait until installation process finishes.
Run Taylor using:

```
taylor -c sdf
```

Play Pacman :)

File: TaylorFramework/Modules/EasterEggs/PacMan/PacMan.swift [hunk 1]
```diff
@@ -85,38 +84,30 @@ struct Generator {
...
-            } else { return "" }
+        while let path = pathsGenerator.next() {
+            if let file = File(path: path) where charNumber < file.contents.characters.count {
+                return file.contents
+            }
         }
-        return file.contents
+        return ""
     }
 }
```
Comment (thelvis4) [inline TaylorFramework/Modules/EasterEggs/PacMan/PacMan.swift]: Can we have here a newline, please? :prettyplease:

**Comment by thelvis4** [comment]
Finished review.

**Comment by S2dentik** [comment]
Added newline 👍 Ready to be looked at

**Comment by thelvis4** [comment]
Great! Thank you! 😉

**Pull request merged**